server/app.js

```diff
@@ -4,8 +4,8 @@ var cookieParser = require('cookie-parser');
 var logger = require('morgan');
 
 var indexRouter = require('./routes/index');
-var usersRouter = require('./routes/users');
-var commentsRouter = require('./routes/comments');
+var usersRouter = require('./routes/UsersRouter');
```

Comment:
it's in the routes folder, so need to have routes again in the filename. stay consistent in your route naming.